aiowebostv/webos_client.py

```diff
@@ -67,25 +68,13 @@ def __init__(
         self.connection: ClientWebSocketResponse | None = None
         self.input_connection: ClientWebSocketResponse | None = None
         self.futures: dict[int, Future[dict[str, Any]]] = {}
-        self._power_state: dict[str, Any] = {}
-        self._current_app_id: str | None = None
-        self._muted: bool | None = None
-        self._volume: int | None = None
-        self._current_channel: dict[str, Any] | None = None
-        self._channel_info: dict[str, Any] | None = None
-        self._channels: list[dict[str, Any]] | None = None
-        self._apps: dict[str, Any] = {}
-        self._extinputs: dict[str, Any] = {}
-        self._system_info: dict[str, Any] = {}
-        self._software_info: dict[str, Any] = {}
-        self._hello_info: dict[str, Any] = {}
-        self._sound_output: str | None = None
+        self.tv_state = WebOsTvState()
+        self.tv_info = WebOsTvInfo()
```

Comment:
I don't think its a problem, but one caution is that consumers of the API could mutate the state externally (not that they couldn't do it before).